server/notification-providers/discord.js

```diff
@@ -37,6 +37,7 @@ class Discord extends NotificationProvider {
             if (heartbeatJSON["status"] === DOWN) {
                 let discorddowndata = {
```

Comment:
Should this be added to the testing requst a few lines above this too?

Reply:
I don't think so...? I looked for usages of `discordtestdata` and don't see it validating against some expected value. I could add it just to be consistent, but I don't see why it would be needed there. Am I missing something?

Reply:
I would prefer if `discordtestdata` would not be different on this one.
This just creates confusion ^^

Reply:
Ok, added